ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -214,15 +212,15 @@ import StoreKit
     }
 
     @objc(makeDeferredPurchase:completionBlock:)
-    static func makeDeferredPurchase(_ startPurchase: DeferredPromotionalPurchaseBlock,
+    static func makeDeferredPurchase(_ startPurchase: StartPurchaseBlock,
                                      completion: @escaping ([String: Any]?, ErrorContainer?) -> Void) {
-        startPurchase { transaction, purchaserInfo, error, userCancelled in
+        startPurchase { transaction, customerInfo, error, userCancelled in
             if let error = error {
                 completion(nil, ErrorContainer(error: error, extraPayload: ["userCancelled": userCancelled]))
-            } else if let purchaserInfo = purchaserInfo,
+            } else if let customerInfo = customerInfo,
                       let transaction = transaction {
                 completion([
-                    "purchaserInfo": purchaserInfo.dictionary,
+                    "customerInfo": customerInfo.dictionary,
```

Comment:
Curious since I'm working on this: does `CustomerInfo.dictionary` use `rawData` underneath?